README.md

```diff
@@ -170,35 +113,105 @@ More information about the
+| triplestore.filter.containers |   A comma-separate list of containers that should be ignored by the indexer  | http://localhost:8080/fcrepo/rest/audit | | 
+| triplestore.namedGraph |  ?  | null | | 
+| triplestore.prefer.include |  ?  | null | | 
+| triplestore.prefer.omit |  ?  | http://www.w3.org/ns/ldp#PreferContainment | | 
 
-This application can be used in conjunction with the Repository
-Re-Indexer to verify the checksums for all Binary resources in
-the repository.
+### LDPath Service
+
+This application implements an LDPath service on repository
+resources. This allows users to dereference and follow URI
+links to arbitrary lengths. Retrieved triples are cached locally
+for a specified period of time.
+
+More information about LDPath can be found at the [Marmotta website](http://marmotta.apache.org/ldpath/language.html).
+
+Note: The LDPath service requires an LDCache backend, such as `fcrepo-service-ldcache-file`.
+
+#### Usage
+The LDPath service responds to `GET` and `POST` requests using any accessible resources as a context.
+
+For example, a request to
+`http://localhost:9086/ldpath/?context=http://localhost/rest/path/to/fedora/object`
+will apply the appropriate ldpath program to the specified resource. Note: it is possible to
+identify non-Fedora resources in the context parameter.
+
+A `GET` request can include a `ldpath` parameter, pointing to the URL location of an LDPath program:
 
-### Repository Serializer
+    `curl http://localhost:9086/ldpath/?context=http://localhost/rest/path/to/fedora/object&ldpath=http://example.org/ldpath`
 
-This application automatically serializes Fedora resources to a filesystem
-location. Using the re-indexer, one can optionally serialize particular
-segments of the repository to a location on the filesystem.
+Otherwise, it will use a simple default ldpath program.
 
-### Repository Re-Indexer
+A `POST` request can also be accepted by this endpoint. The body of a `POST` request should contain
+the entire `LDPath` program. The `Content-Type` of the request should be either `text/plain` or
+`application/ldpath`.
 
-This application allows a user to initiate a re-indexing process
-from any location within the Fedora node hierarchy, sending
-re-indexing requests to a specified list of external applications
-(e.g. fcrepo-indexing-solr, fcrepo-indexing-triplestore, and/or
-fcrepo-fixity).
+    `curl -XPOST -H"Content-Type: application/ldpath" -d @program.txt http://localhost:9086/ldpath/?context=http://localhost/rest/path/to/fedora/object
+
+#### Properties
+| Name      | Description| Default Value |
+| :---      | :---| :----   |
+| fcrepo.cache.timeout | The timeout in seconds for the ldpath cache | 0 |
+| rest.prefix | The LDPath rest endpoint prefix |  no | /ldpath|
+| rest.port| The LDPath rest endpoint port |  no | 9085 |
+| rest.host| The LDPath rest endpoint host |  no | localhost |
+| cache.timeout | LDCache ?  timeout in seconds  |  no | 86400  |
```

Comment:
Why are there two properties do defina a cache timeout?

Reply:
one is for the ldache and one is for the fedora cache.  I'm not completely clear on how those two work together.  I just brought these two properties forward from the previous version.

Reply:
It looks like the fcrepo expiry is for ldclient retrieving resources (see `AbstractHttpProvider` in ldclient-core) and the cache expiry is used by the LDCache during failure (in `LDCache`). So it appears that both are used in different areas, and seems like exposing both is ok.